Romina.Api/Startup.cs

```diff
@@ -19,6 +26,15 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+
+            // Add functionality to inject IOptions<T>
+            services.AddOptions();
+
+            // Add our Config object so it can be injected
+            services.Configure<IProductRepository>(Configuration.GetSection("MySettings"));
```

Comment:
hey bud this isn't quite right .. look at the link I sent you and have a look at what he did differently